sdk/scheduler/src/test/resources/valid-envKey-ports.yml

```diff
@@ -0,0 +1,26 @@
+name: "data-store"
+pods:
+  meta-data:
+    count: 2
+    tasks:
+      meta-data-task:
+        goal: RUNNING
+        cmd: "echo $TASK_NAME >> $TASK_NAME$CONTAINER_PATH_SUFFIX/output && sleep $SLEEP_DURATION"
+        cpus: 0.1
+        memory: 512
```

Comment:
You could drop the `cpus` and `memory` fields, since a valid task spec only needs to have one resource of any type. Then you could do all the casting to `PortRequirement` in one place in the test above.